corehq/apps/dump_reload/couch/load.py

```diff
@@ -47,13 +47,16 @@ def load_objects(self, object_strings, force=False, dry_run=False):
     def _doc_type_matches_filter(self, doc_type):
         return not self.object_filter or self.object_filter.findall(doc_type)
 
-    def _get_db_for_doc_type(self, doc_type):
+    def _get_db_for_doc_type(self, doc_type, chunksize=None):
         if doc_type not in self._dbs:
             couch_db = get_db_by_doc_type(doc_type)
             if couch_db is None:
                 raise DocumentClassNotFound('No Document class with name "{}" could be found.'.format(doc_type))
             callback = LoaderCallback(self._success_counter, self.stdout)
-            db = IterDB(couch_db, new_edits=False, callback=callback)
+            if chunksize:
+                db = IterDB(couch_db, new_edits=False, callback=callback, chunksize=chunksize)
+            else:
+                db = IterDB(couch_db, new_edits=False, callback=callback)
```

Comment:
no need for the if statement:
```suggestion
            db = IterDB(couch_db, new_edits=False, callback=callback, chunksize=chunksize)
```